ingestion/src/metadata/ingestion/ometa/ometa_api.py

```diff
@@ -503,6 +503,16 @@ def list_tags_by_category(self, category: str) -> List[Tag]:
         resp = self.client.get(f"{self.get_suffix(Tag)}/{category}")
         return [Tag(**d) for d in resp["children"]]
 
+    def create_tag_category(self, data):
+        resp = self.client.post(f"/tags", data=data.json())
+        print(resp)
```

Comment:
Could we remove the prints? 🙏

Reply:
Yes, removed those, completely forgot about them.